### ISSUE TITLE
About the improvement compared to the paper

### ISSUE THREAD

**Issue by Yinyf0804** [ppengtang/pcl.pytorch#11]
Hi,
Thank you for your great work!
It puzzles me that the performance in your original paper with VGG16 is 43.5%, but there is another 5.5% increase in this repository.
Is there any tricks besides that you have mentioned? Which parts of this implementation are critical to improve the overall results? 
By the way, do you ever try to use this results as pseudo label and train a fully-supervised network? And could you tell me the performance?

Best Regards.

**Comment by ppengtang** [comment]
Hi, thanks for your interest in our work.
There are two tricks to obtain the mAP 49.2%:
1. As mentioned in the README, we embed the [trick](http://pengtang.xyz/publications/0640-supp.pdf) proposed in our [ECCV paper](http://pengtang.xyz/publications/0640.pdf) for better performance.
2. Keeping more boxes during the mAP evaluation, just as done in some other recent WSOD papers.

Note that the second trick only improves the mAP score and has no influence on the CorLoc score.

I haven't tried to use our new results as pseudo labels to train fully-supervised detectors.

Hope these help you.

**Comment by Yinyf0804** [comment]
Thank you so much!
